package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "xmldom",
+  "name": "@xmldom/xmldom",
```

Comment:
👍 now agreed on my part, should maximize consistency with the GitHub project URL and how this project would normally be installed from GitHub: `npm:xmldom/xmldom` or `npm:xmldom/xmldom#version` (`version` could be branch, tag, or commit ID)